editor.planx.uk/src/@planx/components/Pay/Editor/Editor.tsx

```diff
@@ -69,9 +69,9 @@ const Component: React.FC<Props> = (props: Props) => {
                 <Input
                   format="data"
                   name="fn"
-                  value={values.fn}
+                  value={PASSPORT_FN}
```

Comment:
This should mean that any existing services with a value other and `application.fee.payable` just need the node to be saved to update this.

In reality, these would just be test services so I don't think there's a need for anything more sophisticated here such as a validation check or data migration.

Reply:
Confirmed that a legacy, un-updated Pay component still knows how to read/display correct amount: https://4058.planx.pizza/testing/pay-test :+1: 

And modal looks correct for next "Update"/"Save"